GestureHandler.js

```diff
@@ -55,6 +55,17 @@ UIManager.RCTView.directEventTypes = {
 
 const State = RNGestureHandlerModule.State;
 
+// Used code below as it was impossible to do it with restructuration
+// FIXME to be removed
+Object.defineProperty(State, 'UNDETERMINED', {
```

Comment:
can we limit that to only work in DEV mode? In prod users won't see warnings anyways and we can avoid using proxies which are pretty inefficient on Android right now